### DISCUSSION TITLE
React native + tailwindcss

### DISCUSSION THREAD

**Discussion by filippobrigati** [tailwindlabs/tailwindcss#9919] [Help] [answered]
Hello everyone, i'm learning to use react native, is it possible to use it with tailwind css? if yes, do you just install with npm / yarn package?

**Comment by MattHeslington** [comment, marked as answer]
If you're using Expo, you can use [tailwind-rn](https://github.com/vadimdemedes/tailwind-rn)

**Reply by filippobrigati** [reply]
Okey i'll check, thank you!
